lib/pure/math.nim

```diff
@@ -162,10 +162,23 @@ when defined(js):
   proc toBitsImpl(x: float): array[2, uint32] =
     asm """
     const buffer = new ArrayBuffer(8);
-    const floatBuffer = new Float64Array(buffer);
-    const uintBuffer = new Uint32Array(buffer);
-    floatBuffer[0] = `x`;
-    `result` = uintBuffer
+    const a = new Float64Array(buffer);
+    const b = new Uint32Array(buffer);
+    a[0] = `x`;
+    `result` = b
+    """
+
+  proc jsSetSign(x: float, sgn: bool): float =
```

Comment:
we need more compact JS bit operations.
tobits/frombits can't reuse arraybuffer.

Reply:
i'm now using those procs in this PR